ui/package.json

```diff
@@ -20,7 +20,10 @@
         "prettier --single-quote --trailing-comma es5 --print-width 100 --write",
         "git add"
       ],
-      "ui/app/styles/**/*.*": ["prettier --write", "git add"]
+      "ui/app/styles/**/*.*": [
```

Comment:
npm/yarn and prettier always fight the formatting of these lines. I try to make prettier win since that's easier to keep up with.